src/Components/FilterDropdown/FilterDropdown.js

```diff
@@ -0,0 +1,70 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+
+function FilterDropdown(props) {
+  const [filterItem, setFilterItem] = useState('상담 많은 순');
+  const [filterValid, setFilterValid] = useState(false);
+  const filterItems = [
+    { name: '-sell_count', value: '상담 많은 순' },
+    { name: '-create_at', value: '최근 등록 순' },
+    { name: '-price', value: '가격 높은 순' },
+    { name: 'price', value: '가격 낮은 순' },
+  ];
+
```

Comment:
- 이 부분은 변하지 않는 값 (상수) 이기 때문에, 컴포넌트 바깥에서 선언해주세요! 
- 상수를 네이밍 하실 때는 대문자 스네이크 케이스를 사용해주세요! 대문자 스네이크 케이스를 사용해 주셔야지, 이 변수가 상수인지 스테이트인지 수월하게 구분할 수 있습니다!
```suggestion
  const FILTER_ITEMS = [
    { name: '-sell_count', value: '상담 많은 순' },
    { name: '-create_at', value: '최근 등록 순' },
    { name: '-price', value: '가격 높은 순' },
    { name: 'price', value: '가격 낮은 순' },
  ];

```